settings.gradle.kts

```
@@ -11,7 +11,7 @@ pluginManagement {
         maven(url = "https://maven.minecraftforge.net/")
     }
     plugins {
-        id("com.github.johnrengelman.shadow") version "7.0.0"
+        id("com.github.johnrengelman.shadow") version "7.1.2"
         id("gg.essential.loom") version "0.10.0.3"
```

```suggestion
        id("gg.essential.loom") version "0.10.0.4"
```
This will fix the build failure.